src/main/java/at/hannibal2/skyhanni/features/chat/PowderMiningFilterConfig.java

```diff
@@ -0,0 +1,114 @@
+package at.hannibal2.skyhanni.features.chat;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import com.google.gson.annotations.Expose;
+import io.github.notenoughupdates.moulconfig.annotations.Accordion;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorDropdown;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorSlider;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
+
+public class PowderMiningFilterConfig {
+    @Expose
+    @ConfigOption(name = "Enabled", desc = "Hide messages while opening chests in the Crystal Hollows.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean enabled = false;
+
+    @Expose
+    @ConfigOption(
+        name = "Powder", desc = "Hide §dGemstone §7and §aMithril §7Powder rewards under a certain amount." +
+        "\n§a0§7: §aShow all\n§c20000§7: §cHide all"
+    )
+    @ConfigEditorSlider(minValue = 0, maxValue = 20000, minStep = 500)
+    public int powderFilterThreshold = 1000;
+
+    @Expose
+    @ConfigOption(
+        name = "Essence", desc = "Hide §6Gold §7and §bDiamond §7Essence rewards under a certain amount." +
+        "\n§a0§7: §aShow all\n§c10§7: §cHide all"
+    )
+    @ConfigEditorSlider(minValue = 0, maxValue = 10, minStep = 1)
+    public int essenceFilterThreshold = 3;
+
+    @Expose
+    @ConfigOption(name = "Gemstones", desc = "")
+    @Accordion
+    public PowderMiningGemstoneFilterConfig gemstoneFilterConfig = new PowderMiningGemstoneFilterConfig();
+
+    @Expose
+    @ConfigOption(name = "Ascension Rope", desc = "Hide Ascension Rope rewards.")
+    @ConfigEditorBoolean
+    public boolean ascensionRope = true;
+
+    @Expose
+    @ConfigOption(name = "Wishing Compass", desc = "Hide Wishing Compass rewards.")
+    @ConfigEditorBoolean
+    public boolean wishingCompass = true;
+
+    @Expose
+    @ConfigOption(name = "Oil Barrel", desc = "Hide Oil Barrel rewards.")
+    @ConfigEditorBoolean
+    public boolean oilBarrel = true;
```

Comment:
Just a single though, this here seems a bit ugly and too much, maybe use a list like the Dungeon Filter one?
![grafik](https://github.com/hannibal002/SkyHanni/assets/69345714/77d1c555-095c-45af-bb7d-13e4fe7b0207)

Reply:
Can definitely do that for the boolean ones, gimme 20

Reply:
![image](https://github.com/hannibal002/SkyHanni/assets/40234707/c8653d58-0cf4-440a-b51e-36d58e0ab1a4)

Fixed in latest commit